### PR TITLE
EDU-6514 Feedback Orders API - Update VTEX - Orders API.json

### PR DESCRIPTION
**Pull request by julia-rabello**
[Preview](https://preview.readme.io/reference/GetOrder?url=https%3A%2F%2Fraw.githubusercontent.com%2Fvtex%2Fopenapi-schemas%2FEDU-6514-feedback-orders-api%2FVTEX+-+Orders+API.json)
[EDU-6514](https://vtex-dev.atlassian.net/browse/EDU-6514
)
#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Orders API.json [hunk 1]
```diff
@@ -2182,34 +2192,127 @@
...
+                                                    "title": "message",
+                                                    "description": "Error message",
+                                                    "type": "string",
+                                                    "example": "Acesso não autorizado"
+                                                },
+                                                "exception": {
+                                                    "title": "exception",
+                                                    "description": "Exception",
+                                                    "type": "string",
+                                                    "example": null
```
Comment (karenkrieger) [inline VTEX - Orders API.json:2237]:
```suggestion
                                                    "example": null,
                                                    "nullable": true
```

File: VTEX - Orders API.json [hunk 2]
```diff
@@ -2182,34 +2192,127 @@
...
+                                                    "title": "message",
+                                                    "description": "Error message",
+                                                    "type": "string",
+                                                    "example": "Order status should be ready-for-handling to perform this action"
+                                                },
+                                                "exception": {
+                                                    "title": "exception",
+                                                    "description": "Exception",
+                                                    "type": "string",
+                                                    "example": null
```
Comment (karenkrieger) [inline VTEX - Orders API.json:2288]:
```suggestion
                                                    "example": null,
                                                    "nullable": true
```

File: VTEX - Orders API.json [hunk 3]
```diff
@@ -3254,7 +3373,9 @@
         },
         "/api/oms/pvt/feed/orders/status": {
             "get": {
-                "tags": ["Feed v2 (deprecated)"],
+                "tags": [
+                    "Feed v2 (deprecated)"
+                ],
                 "summary": "Get feed order status",
                 "description": "",
```
Comment (karenkrieger) [inline VTEX - Orders API.json:3380]: Adicionar descrição

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Orders API.json (hunk 3 above)
Comment (julia-rabello) [inline VTEX - Orders API.json:3380]:
```suggestion
                "description": "Get feed order status (deprecated)",
```

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**